quipucords/api/scanjob/model.py

```diff
@@ -280,6 +267,28 @@ def queue(self):  # noqa: C901
         self.save()
         self.log_current_status()
 
+    def delete_inspect_results(self):
+        """Delete InspectResults exclusively related to this ScanJob."""
+        # results_from_other_jobs = InspectResult.objects.exclude(
+        #     tasks__in=self.tasks.all()
+        # ).values(
+        #     "id",
+        # )
+        # InspectResult.objects.exclude(id__in=results_from_other_jobs).filter(
+        #     tasks__job_id=self.id
+        # ).delete()
+        # TODO: why the approach above didn't work? I had to use the m2m model to do it
```

Comment:
I need to drop this commented code, but first... Does anyone see a way to make this deletion work without using through model like below?